### PR TITLE
Feature/more install

### PR DESCRIPTION
**Pull request by jplouis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#zendev
+
+##Description
+zendev takes care of setting up and managing the disparate components of a standard Zenoss development environment, 
+with or without the control plane. It's primary usecase is to checkou out all github repositories needed for
```
Comment (tgjonestx) [inline README.md]: "with or without the control plane" - maybe "with or without Control Center"?

also, "chekou out" should be "checkout"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+#zendev
+
+##Description
+zendev takes care of setting up and managing the disparate components of a standard Zenoss development environment, 
+with or without the control plane. It's primary usecase is to checkou out all github repositories needed for 
+development and to build a developer focused docker image to run Zenoss in Control-Center. It can also:
+
+* Add GitHub repositories to your source environment
```
Comment (tgjonestx) [inline README.md]: We removed `zendev add`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+* Quickly view branches across some or all repositories
+* Perform arbitrary actions across some or all repositories
```
Comment (tgjonestx) [inline README.md]: I don't think zendev2 supports this any longer

**Review by tgjonestx — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jplouis) [inline README.md]: jig supports it, but I'll remove the doc

**Pull request merged**